code/controllers/subsystem/persistence.dm

```diff
@@ -0,0 +1,109 @@
+#define FILE_RECENT_MAPS "data/RecentMaps.json"
+
+#define KEEP_ROUNDS_MAP 3
+
+//Season defines, universal for all seasons
+#define LAST_UPDATE "last_update" //last time the season changed
+#define CURRENT_SEASON "current_season" //current season of the section
+
+//Season names
+#define SEASONAL_GUNS "seasonal_guns"
+
+SUBSYSTEM_DEF(persistence)
+	name = "Persistence"
+	init_order = INIT_ORDER_PERSISTENCE
+	flags = SS_NO_FIRE
+
+	///Stores how long each season should last
+	var/list/seasons_durations = list(
+		SEASONAL_GUNS = 0.001 DAYS,
```

Comment:
that's for testing i suppose

Reply:
Yes, I may remove and add this later or hopefully have some buckets I could add with this

Reply:
added gun buckets that rotate every 4 days